infrastructure/dynamodbkutter.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
maybe name the resource also kutter table. It is easier to identify than

Reply:
OK :)

Reply:
I have changed it and pushed it